themes/bootstrap3/templates/search/history.phtml

```diff
@@ -55,12 +55,7 @@
 
 <?php if ($saveSupported): ?>
   <div class="<?=$this->layoutClass('sidebar')?>" id="myresearch-sidebar">
-    <?=
-      $this->context($this)->renderInContext(
-          "myresearch/menu.phtml",
-          ['active' => 'history']
-      );
-    ?>
+    <?=$this->context($this)->renderInContext("myresearch/menu.phtml", ['active' => 'history'])?>
```

Comment:
I see that in a couple of places, the code has been collapsed like this. I don't object to this change since it makes things more concise while (in my opinion, at least) remaining reasonably readable, but I'm curious if it has anything to do with other changes, or whether it somehow happened unintentionally.

Reply:
This is a totally valid question. Normally I tend to break up long lines, especially parameter-fests like those types, but it seems unnecessary in this specific instance. But I actually don't posses any memory of having done this, its a bit weird, for a pull request i don't want to change things without explicit reason. I take it its fine as it is?